src/chunk.rs

```diff
@@ -47,6 +47,12 @@ impl Chunk {
             .extend(other.offsets.iter().map(|&(id, offset)| (id, base + offset)));
     }
 
+    /// An iterator over the references of the top-level objects
+    /// of the chunk, in the order they appear in the chunk.
+    pub fn object_refs(&self) -> impl IntoIterator<Item = Ref> + '_ {
+        self.offsets.iter().map(|o| o.0)
+    }
```

Comment:
```suggestion
    pub fn refs(&self) -> impl IntoIterator<Item = Ref> + '_ {
        self.offsets.iter().map(|&(id, _)| id)
    }
```
I think just `refs` suffices. The destructruing thing is purely personal preference. I find it a bit more readable at a glance because `o.0` doesn't tell me what it is.